kernel/Dockerfile

```diff
@@ -98,8 +101,14 @@ RUN make defconfig && \
     cp System.map /out && \
     ([ -n "${DEBUG}" ] && cp vmlinux /out || true)
 
+# WireGuard
+RUN git clone --depth 1 -b "$WIREGUARD_TAG" "$WIREGUARD_REPO" /wireguard && \
```

Comment:
release tarballs are great if they can be verified. We also do that in a couple of places (including in this Dockerfile :). I'd be happy with that